### PR TITLE
[STHLM-2019] Add new members to stockholm team

### PR DESCRIPTION
**Pull request by JasmineCrozier**
I added 3 persons to our team

### CODE REVIEW

**Comment by phrawzty** [comment]
I thought that I resolved the conflict: 
```
CONFLICT (rename/add): Rename data/events/2018-stockholm.yml->data/events/2019-stockholm.yml in HEAD. data/events/2019-stockholm.yml added in 1d8b4cf532ff19512dada65011bf3202707b2351
```
But somehow all I apparently managed to do is ignore your commit, maybe? I'm not entirely sure. Raising on Slack.

**Comment by JasmineCrozier** [comment]
not sure what happened here...
I thought that I only added these 3 people to Team_members:
  - name: "Jonas Linde"
  - name: "Mark Baker"
    employer: "EA"
    twitter: "markltbaker"
  - name: "Tony Collander"
    employer: "Valtech"
    twitter: "tonycollander"

**Comment by phrawzty** [comment]
>  I thought that I only added these 3 people to Team_members:

If you look at your [commit](https://github.com/devopsdays/devopsdays-web/pull/5954/commits/1d8b4cf532ff19512dada65011bf3202707b2351), it contains quite a bit more than that. It might be easier to just to close this and try again.

My advice: do a _fresh_ pull and make sure that you're branching off `master`, then make a single commit with the organiser update, and open a new PR.

File: data/events/2019-stockholm.yml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+
+# These are the same people you have on the mailing list and Slack channel.
+team_members: # Name is the only required field for team members.
+  - name: "Anders Bruvik"
+    twitter: "bruvik"
+    employer: "Safespring"
+  - name: "Emily Bache"
+    twitter: "emilybache"
+    employer: Praqma
```
Comment (MikeRosTX) [inline data/events/2019-stockholm.yml]: If I would take a wild guess, I would add the **" "** (**_double quotes_**) around the 3 organizers from **_Pragma_** that are missing.

Comment (phrawzty) [reply data/events/2019-stockholm.yml]: That might be _an_ issue, but it's not _the_ issue. See the change list in 
1d8b4cf:

image

**Pull request closed without merging**